src/pseudo_intrinsics.h

```diff
@@ -490,35 +555,14 @@ typedef __m64i vtype;
 #endif /* __SSE3__ || __MIC__ */
```

Comment:
BTW please not that the ternary function is supposed to be optimized away completely, so the code will be just either of the functions in each place this macro is used. Maybe you didn't realize that.